pytest_inmanta/plugin.py

```diff
@@ -1137,6 +1137,14 @@ def enumerate_model(model: str):
         for key, blob in exporter._file_store.items():
             self.add_blob(key, blob)
 
+        for key, value in inmanta.resources.resource._resources.items():
+            name_value = value[1].get("name")
+            if name_value == "id":
+                warnings.warn(
+                    f"In the next release it will not be possible anymore to use an id_attribute called id for {key}",
```

Comment:
It's not uncommon that we wait a few major releases until we actually remove support for a deprecated feature.
```suggestion
                    f"In one of the next major releases of inmanta-core it will not be possible anymore to use an id_attribute called id for {key}",
```